rest/rest-api/src/main/java/org/apache/brooklyn/rest/api/ActivityApi.java

```diff
@@ -57,6 +57,18 @@ public TaskSummary get(
             @ApiParam(value = "Task ID", required = true) @PathParam("task") String taskId);
 
     @GET
+    @Path("/{task}/children/recurse")
+    @ApiOperation(
+            value = "Fetch all child tasks details",
+            response = org.apache.brooklyn.rest.domain.TaskSummary.class,
+            responseContainer = "map")
```

Comment:
Looks like that's not a valid value - http://docs.swagger.io/swagger-core/apidocs/com/wordnik/swagger/annotations/ApiOperation.html#responseContainer()